sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -59,17 +60,25 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
             Argument.AssertNotNull(credential, nameof(credential));
             Argument.AssertNotNull(options, nameof(options));
 
+            string scope = options.Audience + "/.default";
+
+            //if no Audience is provided, default to Azure Public Cloud
+            if (options.Audience == null)
+            {
+                scope = TextAnalyticsAudience.AzureResourceManagerPublicCloud + "/.default";
```

Comment:
The `/.default` is already included in the `TextAnalyticsAudience.AzureResourceManagerPublicCloud` value.